stories/features/newTab/components/clock/index.tsx

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import {
+   StyledClock,
+   StyledPeriod,
+   StyledTime,
+   StyledTimeSeparator
+ } from './style'
+
+export interface ClockTheme {
+  color?: string
+}
+export interface ClockProps {
+  id?: string
+  customStyle?: ClockTheme
```

Comment:
I don't think we need `customStyle` since it's not passed through, and not used, right @cezaraugusto ?

Reply:
ya left-over, removed